tools/psyq-obj-parser/psyq-obj-parser.cc

```diff
@@ -1185,7 +1187,7 @@ Usage: {} input.obj [input2.obj...] [-h] [-v] [-d] [-n] [-p prefix] [-o output.o
                 }
             }
         }
-        delete file;
+        file.reset();
```

Comment:
if you don't make the above change this line is redundant since the scope of `file` ends here